google/cloud/spanner_v1/streamed.py

```diff
@@ -147,6 +148,17 @@ def __iter__(self):
             except StopIteration:
                 return
 
+    def to_dataframe(self):
+        """Returns the response in a pandas dataframe of the StreamedResultSet object
+
+        :rtype: pandas.DataFrame
+
+        :returns: Dataframe created from the StreamedResultSet response object returned by execute_sql() method
```

Comment:
The context of this docstring is in the StreamedResultSet class, so we don't have to repeat tha6t.

```suggestion
        :returns: DataFrame created from the result set
```